### PR TITLE
Add static provisioning example to PowerFlex

### PR DESCRIPTION
**Pull request by falfaroc**
# Description
It was identified that documentation for creating static provisioning on PowerFlex was missing. Similar to other drivers that support this feature, a section for the documentation was added for consistency. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/350 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/features/powerflex.md [hunk 1]
```diff
@@ -522,6 +522,66 @@ Then run:
 this test deploys the pod with two ephemeral volumes, and write some data to them before deleting the pod.   
 When creating ephemeral volumes, it is important to specify the following within the volumeAttributes section: volumeName, size, storagepool, and if you want to use a non-default array, systemID.  
 
+## Consuming Existing Volumes with Static Provisioning
+
+You can use existent volumes from PowerFlex array as Persistent Volumes in your Kubernetes, perform the following steps:
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/powerflex.md:527]: Change "the following" to "these"

File: content/docs/csidriver/features/powerflex.md [hunk 2]
```diff
@@ -522,6 +522,66 @@ Then run:
...
+1. Open your volume in the PowerFlex Management UI. Filter by just your volume and the volume ID can be located in the URL.
+2. Keep track of your system ID and volume ID as they will be needed in the following steps.
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/powerflex.md:529]: Replace "in the following steps" with "during this procedure"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There are a couple of minor edits.

File: content/docs/csidriver/features/powerflex.md (hunk 2 above)
Comment (falfaroc) [reply content/docs/csidriver/features/powerflex.md]: Updated.

File: content/docs/csidriver/features/powerflex.md [hunk 3]
```diff
@@ -522,6 +522,66 @@ Then run:
 this test deploys the pod with two ephemeral volumes, and write some data to them before deleting the pod.   
 When creating ephemeral volumes, it is important to specify the following within the volumeAttributes section: volumeName, size, storagepool, and if you want to use a non-default array, systemID.  
 
+## Consuming Existing Volumes with Static Provisioning
+
+You can use existent volumes from PowerFlex array as Persistent Volumes in your Kubernetes, perform these steps:
+1. Open your volume in the PowerFlex Management UI. Filter by just your volume and the volume ID can be located in the URL.
+2. Keep track of your system ID and volume ID as they will be needed during this procedure.
+3. Create PersistentVolume and use this volume ID in the volumeHandle with the format <systemID>-<volumeID> in the manifest. Modify other parameters according to your needs.
```
Comment (bharathsreekanth) [inline content/docs/csidriver/features/powerflex.md:530]: I would just add an ex. of the volume handle. maybe use what you used and change it slightly so as to be not real.

**Review by bharathsreekanth — APPROVED** [review]

File: content/docs/csidriver/features/powerflex.md [hunk 4]
```diff
@@ -522,6 +522,66 @@ Then run:
 this test deploys the pod with two ephemeral volumes, and write some data to them before deleting the pod.   
 When creating ephemeral volumes, it is important to specify the following within the volumeAttributes section: volumeName, size, storagepool, and if you want to use a non-default array, systemID.  
 
+## Consuming Existing Volumes with Static Provisioning
+
+You can use existent volumes from PowerFlex array as Persistent Volumes in your Kubernetes, perform these steps:
```
Comment (bharathsreekanth) [inline content/docs/csidriver/features/powerflex.md:527]: minor. I would just rephrase as "To use existing volumes from PowerFlex array as Peristent volumes in your Kubernetes environment, perform these steps:"

**Review by bharathsreekanth — APPROVED** [review]

**Comment by bharathsreekanth** [comment]
> 

Good point Christian. Could you provide the scli cmd you used, we can add that. PowerFlex UI is not as intuitive or powerful as PowerStore UI.

**Review by JacobGros — DISMISSED** [review]

**Comment by ChristianAtDell** [comment]
> > 
> 
> Good point Christian. Could you provide the scli cmd you used, we can add that. PowerFlex UI is not as intuitive or powerful as PowerStore UI.

Credit goes to Don Khan for finding these cmds out:

scli --mdm_ip <IPs, comma separated> --login --username <user> --password <pass>
scli --vol --query_volume --volume_name <name>

I do recommend expanding on the UI version's directions as well, though, for ease of use.

**Comment by bharathsreekanth** [comment]
> > > 
> > 
> > 
> > Good point Christian. Could you provide the scli cmd you used, we can add that. PowerFlex UI is not as intuitive or powerful as PowerStore UI.
> 
> Credit goes to Don Khan for finding these cmds out:
> 
> scli --mdm_ip <IPs, comma separated> --login --username --password scli --vol --query_volume --volume_name
> 
> I do recommend expanding on the UI version's directions as well, though, for ease of use.

I looked around in the UI and I was not able to find a way to get the Id.

**Comment by falfaroc** [comment]
@ChristianAtDell @bharathsreekanth The only way I was able to get the volume ID through the UI is by clicking the volume in the topology section which filters for only that volume. Afterwards, in the URL, the volume UUID is present.

image

image

I will update the documentation to include both the `scli` commands and the `ui` approaches.

**Comment by bharathsreekanth** [comment]
> @ChristianAtDell @bharathsreekanth The only way I was able to get the volume ID through the UI is by clicking the volume in the topology section which filters for only that volume. Afterwards, in the URL, the volume UUID is present.
> 
> image
> 
> image
> 
> I will update the documentation to include both the `scli` commands and the `ui` approaches.

Thanks Fernando.  It is not really easy for one to know that is the ID of the volume. I suppose we could add a footnote below the documentation to tell how to get the ID.  I am not a fan of it, but I dont see any easy way. Thoughts?

**Comment by donatwork** [comment]
> > @ChristianAtDell @bharathsreekanth The only way I was able to get the volume ID through the UI is by clicking the volume in the topology section which filters for only that volume. Afterwards, in the URL, the volume UUID is present.
> > image
> > image
> > I will update the documentation to include both the `scli` commands and the `ui` approaches.
> 
> Thanks Fernando. It is not really easy for one to know that is the ID of the volume. I suppose we could add a footnote below the documentation to tell how to get the ID. I am not a fan of it, but I dont see any easy way. Thoughts?

I'm not a fan of the URL approach. That may be useful as a hint or note but not as main line documentation. We should just document the scli option as a PowerFlex user should already be familiar with scli. Put the UI hint in a note.

**Review by bharathsreekanth — APPROVED** [review]

**Comment by bharathsreekanth** [comment]
I did confirm with Powerflex engg team that there is no easy from UI to get this. Let us rely on the scli for now.

**Comment by donatwork** [comment]
> I did confirm with Powerflex engg team that there is no easy from UI to get this. Let us rely on the scli for now.

Can we open an enhancement request so that the ID is available in the UI.

**Comment by bharathsreekanth** [comment]
> > I did confirm with Powerflex engg team that there is no easy from UI to get this. Let us rely on the scli for now.
> 
> Can we open an enhancement request so that the ID is available in the UI.

I asked them that. waiting to hear back

**Review by JacobGros — DISMISSED** [review]

File: content/docs/csidriver/features/powerflex.md [hunk 5]
```diff
@@ -522,6 +522,72 @@ Then run:
 this test deploys the pod with two ephemeral volumes, and write some data to them before deleting the pod.   
 When creating ephemeral volumes, it is important to specify the following within the volumeAttributes section: volumeName, size, storagepool, and if you want to use a non-default array, systemID.  
 
+## Consuming Existing Volumes with Static Provisioning
+
+To use existing volumes from PowerFlex array as Peristent volumes in your Kubernetes environment, perform these steps:
+1. Log into one of the MDMs of the PowerFlex cluster.
+2. Execute the following commands to retrieve the `systemID` and `volumeID`.
```
Comment (rsedlock1958) [inline content/docs/csidriver/features/powerflex.md:529]: Change "the following" to "these"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There is one minor edit.

**Review by rsedlock1958 — APPROVED** [review]
This looks good. No edits required.

**Review by abhi16394 — APPROVED** [review]

**Review by JacobGros — APPROVED** [review]

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. No edits are reqired.

**Review by JacobGros — APPROVED** [review]

**Review by jooseppi-luna — APPROVED** [review]

**Review by mjsdell — APPROVED** [review]

**Pull request merged**